### PR TITLE
Code Review Request: DESIGN TEARDOWN

### PR DESCRIPTION
**Pull request by cgcarlosg**
This project is done according to the requirements of the fifth Microverse project. 
It is an excellent exercise to train to think in terms of visual hierarchy, typography, and design principles.
In this project, I used grayscale and worked with Grid, shapes, and contrast.
Everything was successfully run in the Linters workflow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+            <div class="footer-four"></div>
+            <div class="footer-five"></div>
+            <div class="footer-six"></div>
+            <div class="footer-seven"></div>
+            <div class="footer-eight"></div>
+            <div class="footer-nine"></div>
+            <div class="footer-ten"></div>
+            <div class="footer-eleven"></div>
+ 
+        </div>
```
Comment (umairarshadbutt) [inline index.html:485]: A better approach is using `<ul>` and `<li>`
Please add centered links in `<li>`

**Review by umairarshadbutt — COMMENTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @cgcarlosg
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly work on the suggested changes 👇.

- [ ] Please add more description in pull request( [screenshot](https://user-images.githubusercontent.com/29689273/96525354-1ca57f80-1294-11eb-8b9f-f883a522a0ce.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] Change footer menus to `<ul>` with `<li>`. This is more semantic than the `<div>`. See inline comments
- [ ] Remove the horizontal scroll bar (You can visit [image](https://user-images.githubusercontent.com/29689273/96524833-c126c200-1292-11eb-9a18-1d05de534b7c.png))
- [ ] **[OPTIONAL]** Use the navbar tag (`nav`) for navbar

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by Temmarie — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @cgcarlosg ! 
### You have done well so far :clap: However, you did not finish all the previous review changes. Kindly work on them and submit them for another code review.

Your project looks great. However, your project cannot be approved until you write a descriptive PR Message. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. Please refer to [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more information.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️  

### Hi @cgcarlosg!  

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 🎉 

### You may merge, good luck with your next projects! :rocket: 
### Happy Coding! 👍 
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Comment by cgcarlosg** [comment]
project completed

**Pull request closed without merging**